common/src/main/java/com/wynntils/mc/mixin/AbstractContainerScreenMixin.java

```diff
@@ -34,6 +35,17 @@ private void renderPost(PoseStack client, int mouseX, int mouseY, float partialT
                 (AbstractContainerScreen<?>) (Object) this, client, mouseX, mouseY, partialTicks, this.hoveredSlot));
     }
 
+    @Inject(method = "renderLabels(Lcom/mojang/blaze3d/vertex/PoseStack;II)V", at = @At("HEAD"), cancellable = true)
+    private void renderLabels(PoseStack poseStack, int mouseX, int mouseY, CallbackInfo ci) {
+        ContainerRenderLabelEvent event =
```

Comment:
I would love to see this an event that you can modify the rendered text, with `WrapOperation` like `ItemTooltipRenderEvent`. However that is a much more complex mixin. I can write it for you, or you can go on the adventure of figuring that out for yourself. (If it is too messy because of bad MC source code, let me know)

Reply:
(Just checked the source, this is rather easy if you know what you do.)

Reply:
Having trouble getting it working, feel free to add it whenever you're free and I'll just look at how you've done it to see what I was doing wrong

Reply:
There you go. Ping me on discord if you don't understand parts of it. https://github.com/Wynntils/Artemis/pull/2023/commits/11b7924e7da5ae22fb8352bc555062dd08afa86f